src/cordova/apple/xcode/ios/PaperVPN.xcodeproj/project.pbxproj

```diff
@@ -1054,15 +1060,15 @@
 				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
-				INFOPLIST_FILE = "$(SRCROOT)/Outline/VpnExtension-Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/PaperVPN/VpnExtension-Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 					"@executable_path/../../Frameworks",
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = org.outline.ios.client.VpnExtension;
-				"PRODUCT_BUNDLE_IDENTIFIER[sdk=macosx*]" = org.outline.macos.client.VpnExtension;
+				PRODUCT_BUNDLE_IDENTIFIER = io.paperpaper.xray.ios.client.VpnExtension;
```

Comment:
we are building io.papervpn.client now
see https://github.com/paperpaperru/papervpn/commit/2f6d2539bafa3d08ad8cbc5c0f4b88b7bab90b63